core/src/main/java/com/github/kaktushose/jda/commands/definitions/interactions/InteractionRegistry.java

```
@@ -183,7 +206,7 @@ public <T extends Definition> T find(Class<T> type, boolean internalError, Predi
                 .filter(predicate)
                 .findFirst()
                 .orElseThrow(() -> internalError
-                        ? new IllegalStateException("No interaction found! Please report this error to the devs of jda-commands.")
+                        ? new JDACException.Internal("No interaction found! Please report this error to the devs of jda-commands.")
```

"Please report this error to the devs of jda-commands." can be removed, would be duplicated